.github/workflows/test-e2e-cli.yml

```diff
@@ -9,75 +9,35 @@ on:
-    ]
-  TEST_PROJECTS_WINDOWS: |
-    [
-      {"host":"connection.keboola.com","project":8778,"stagingStorage":"s3","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_8778_TOKEN }}", "legacyTransformation": true},
-      {"host":"connection.keboola.com","project":8779,"stagingStorage":"s3","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_8779_TOKEN }}", "legacyTransformation": true},
-      {"host":"connection.keboola.com","project":8780,"stagingStorage":"s3","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_8780_TOKEN }}", "legacyTransformation": true},
-      {"host":"connection.keboola.com","project":8782,"stagingStorage":"s3","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_8782_TOKEN }}", "legacyTransformation": true},
-      {"host":"connection.europe-west3.gcp.keboola.com","project":49,"stagingStorage":"gcs","backend":"bigquery","token":"${{ secrets.TEST_KBC_PROJECT_49_TOKEN }}"},
-      {"host":"connection.europe-west3.gcp.keboola.com","project":50,"stagingStorage":"gcs","backend":"bigquery","token":"${{ secrets.TEST_KBC_PROJECT_50_TOKEN }}"},
-      {"host":"connection.north-europe.azure.keboola.com","project":10894,"stagingStorage":"abs","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_10894_TOKEN }}", "legacyTransformation": true},
-      {"host":"connection.north-europe.azure.keboola.com","project":10902,"stagingStorage":"abs","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_10902_TOKEN }}", "legacyTransformation": true},
-      {"host":"connection.north-europe.azure.keboola.com","project":10903,"stagingStorage":"abs","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_10903_TOKEN }}", "legacyTransformation": true},
-      {"host":"connection.north-europe.azure.keboola.com","project":10904,"stagingStorage":"abs","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_10904_TOKEN }}", "legacyTransformation": true}
-    ]
-
 jobs:
   test:
     name: ${{ matrix.name }}
-    concurrency:
-      group: tests-e2e-cli-${{ matrix.name }}
-      cancel-in-progress: false
     strategy:
       fail-fast: false
       matrix:
         include:
           - name: linux
             os: ubuntu-latest
-            projects: TEST_PROJECTS_LINUX
           - name: mac-os
             os: macos-latest
-            projects: TEST_PROJECTS_MACOS
           - name: windows
             os: windows-2022
-            projects: TEST_PROJECTS_WINDOWS
     runs-on: ${{ matrix.os }}
     steps:
       - name: Setup line endings
         run: git config --global core.autocrlf false
-
+      
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Set TEST_KBC_PROJECTS
+        uses: ./.github/actions/export-kbc-projects
+        with:
+          secrets: ${{ toJSON(secrets) }}
+          projects: ${{ vars.TEST_KBC_PROJECTS }}
```

Comment:
This inputs are same in all cases, so maybe we can remove them (?).

Reply:
When you have reusable workflow which we do, we have to have `inputs` field where we insert these variables. I had it without any of these and the reusable workflow did not see these variables and secrets

Reply:
Ok, ... there is also alternative `secrets: inherit`

Reply:
As we have discussed we can leave it like that. It is the simillar to the `secrets: inherit`